### PR TITLE
frontend: add prettier config, instructions in README, also format everything

### PR DESCRIPTION
**Pull request by akhilnarang**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,5 +46,10 @@
       "last 1 firefox version",
       "last 1 safari version"
     ]
+  },
+  "prettier":{
+      "trailingComma": "es5",
+      "tabWidth": 4,
+      "semi": false
```
Comment (ksdfg) [inline package.json]: this is for adding semicolons to the end of all lines, right? true karna better rahega imo

**Review by ksdfg — DISMISSED** [review]
Seems sane enough to me, now I can never open VS Code in my life. File Watchers ftw!

**Review by ksdfg — DISMISSED** [review]
Seems sane enough to me, now I can never open VS Code in my life. File Watchers ftw!

**Review by adityavdesai — DISMISSED** [review]

**Review by ksdfg — DISMISSED** [review]
k, this was it, I promise. This is as readable as black code to me now.

**Review by ksdfg — APPROVED** [review]
k, this was it, I promise. This is as readable as black code to me now.

**Review by adityavdesai — APPROVED** [review]

**Pull request merged**